### DISCUSSION TITLE
Question About Label

### DISCUSSION THREAD

**Discussion by davidgrohl** [h2zero/NimBLE-Arduino#381] [Q&A]
Hi,
is there a way to change this description when paired in windows?

![IMG_0464](https://user-images.githubusercontent.com/17146264/161548619-f3ac3f5d-d4e3-48f7-b275-e28b06b7c9b3.jpeg)

Thanks for help

**Comment by h2zero** [comment]
You could try setting the appearance to something else.

**Reply by davidgrohl** [reply]
Could you give me more specific indications? Where can I set appereance?

**Reply by h2zero** [reply]
Sorry, I totally forgot that I didn't reply. You can set the appearance in the advertising: https://h2zero.github.io/NimBLE-Arduino/class_nim_b_l_e_advertising.html#aa0d30dfb57670cc8180e17dffddad416

**Reply by davidgrohl** [reply]
No problem mate. I’ll give it a try. Thanks for now!
